src/components/providers/ProviderView.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { Box, Stack, Avatar, Text, useTheme, Badge } from "@chakra-ui/react";
+import { useNavigate } from "react-router-dom";
+
+const ProviderView = ({ id, name, title, bio, availability, image }) => {
+  const theme = useTheme();
+  const navigate = useNavigate();
+
+  return (
+    <Box
+      bgColor={theme.colors.neutral[0]}
+      _hover={{
+        bgColor: theme.colors.neutral[300],
+        cursor: "pointer",
+      }}
+      w="full"
+      rounded="md"
+      py={3}
+      px={6}
+      overflow="hidden"
+      maxH="167px"
+      onClick={() => {
+        navigate(`/provider/profile/${id}`);
+      }}
+    >
+      <Stack direction={"row"} spacing={4} align={"center"}>
+        <Avatar
+          w="56px"
+          h="56px"
+          src={image ? image : "https://bit.ly/dan-abramov"}
+          alt={name}
+          borderWidth={1}
+          borderColor={theme.colors.neutral[300]}
+        />
+        <Stack direction={"column"} spacing={0} fontSize={"sm"}>
+          <Text
+            fontWeight={500}
+            color={theme.colors.neutral[900]}
+            fontSize="16px"
+            noOfLines={1}
+          >
+            {name}, {title}
+          </Text>
+          <Text
+            fontWeight={400}
+            color={theme.colors.neutral[700]}
+            fontSize="12px"
+          >
+            Registered Social Worker
+          </Text>
+        </Stack>
+      </Stack>
+      <Stack my="10px">
+        <Text
+          color={theme.colors.neutral[700]}
+          fontWeight={400}
+          fontSize="14px"
+          noOfLines={2}
+        >
+          {bio}
+        </Text>
+      </Stack>
+      <Box d="flex" alignItems="baseline">
+        <Badge
+          rounded="full"
+          px="12px"
+          py="6px"
+          fontWeight={600}
+          fontSize="12px"
+          color={theme.colors.neutral[800]}
+          bgColor={theme.colors.secondary[200]}
+        >
+          Available {availability}
```

Comment:
I think theres a text transform on this element, and the text doesn't match the copy provided

Your app
![CleanShot 2022-04-13 at 17 08 27](https://user-images.githubusercontent.com/19275035/163270637-63df7074-e42e-4480-a0fe-5b6256fa542b.png)

Designs
![CleanShot 2022-04-13 at 17 08 39](https://user-images.githubusercontent.com/19275035/163270666-4012dd9d-5c9c-4483-a469-c8368c4c3cbd.png)